kotlin-inject-compiler/core/src/main/kotlin/me/tatarka/inject/compiler/TypeResultResolver.kt

```diff
@@ -190,6 +190,15 @@ class TypeResultResolver(private val provider: AstProvider, private val options:
         return paramsWithName
     }
 
+    /**
+     * Converts platform nullability to non-null. It's only viable for parameters in java code, never for provided types
+     */
+    private fun withPlatformNullabilityAsNotNull(type: AstType) = if (type.isPlatform()) {
```

Comment:
minor: a function seems overkill since this is only used in the one place (well there's also resolveParamsLegacy but that errors out now and should be removed for the next release anyway)

Reply:
ok, removed function